src/ui/shared/requests/useWalletParams.ts

```diff
@@ -1,20 +1,24 @@
 import { useMemo } from 'react';
 import type { ExternallyOwnedAccount } from 'src/shared/types/ExternallyOwnedAccount';
 
+export function getWalletParams(
```

Comment:
Getting wallet params itself is not enough, we also need to acceptOrigin the moment we click the link.
See ActionButtonRow for reference

Reply:
I believe this comment is related to the RewardsLinkIcon (I'll better renamed it though)